lib/kwtool/crawler/queries/keyword_result_query.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Kwtool.Crawler.Queries.KeywordResultQuery do
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Crawler.Schemas.{Keyword, KeywordResult}
+
+  def search_query(user_id, params) do
```

Comment:
Does this function have a bug if the keyword has multiple keyword results that match the filter? For example, the keyword `car` contains 2 results with URL "car1.com" and "car2.com" if given `car` in url param, would this function return two records of the `car` keyword? 🤔

Reply:
This is very good catch @hanam1ni  https://github.com/byhbt/kwtool/pull/49#discussion_r837340902 :smile: 

the database is designed 1->M (keyword and keyword_result)
1 Keyword can have multiple keyword result.

` if given car in url param, would this function return two records of the car keyword? ` - yes

![image](https://user-images.githubusercontent.com/948856/160810784-452db025-45e5-4871-b229-6d47187d7a8e.png)

Reply:
Fixed by adding the `distint` in https://github.com/byhbt/kwtool/pull/49/commits/b661d4a02195ebae0bc8e488667b23058f3fa5f4